packages/edit-site/src/components/template-part-converter/convert-to-template-part.js

```diff
@@ -119,3 +198,31 @@ export default function ConvertToTemplatePart( { clientIds, blocks } ) {
 		</BlockSettingsMenuControls>
 	);
 }
+
+import { footer, header, layout, check } from '@wordpress/icons';
+const AREA_OPTIONS = [
+	{
+		description: __(
+			'General templates often perform a specific role like displaying post content, and are not tied to any particular area.'
```

Comment:
How would we feel about consolidating `AREA_OPTIONS` with the template part variations updates we made?

https://github.com/WordPress/gutenberg/blob/2c491ad941060b5076bc43093ae7f81788d5b848/packages/block-library/src/template-part/variations.js#L9-L30

Reply:
Great observation!

I actually just pushed some commits that define these descriptions in the php files where we defined the default template part areas.  That is then added to settings which is loaded into the core/editor store, where a selector maps the proper icons to each area.  The new selector `__experimentalGetTemplatePartAreas` in the `core/editor` store could then be used in these other places like you mention so things are consolidated and consistent (block variations, block inspector 'area' selection options, etc.)

I would prefer holding off connecting that selector to these other sections for a follow up though. 😁

Reply:
> I would prefer holding off connecting that selector to these other sections for a follow up though. 😁

Yep! Sounds good